2018-11-20 10:44:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DELETE FROM MAGASIN WHERE ID[*] == 3 "; SQL statement:
DELETE FROM MAGASIN WHERE id == 3 [42000-190]
2018-11-20 10:44:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DELETE FROM MAGASIN WHERE ID[*] == 3 "; SQL statement:
DELETE FROM MAGASIN WHERE id == 3 [42000-190]
2019-01-05 15:45:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 15:45:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 18:08:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 18:09:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 18:09:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 19:06:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 19:15:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 19:34:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 20:42:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 20:59:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 23:02:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 23:04:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-05 23:07:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-06 11:04:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-06 11:06:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-06 11:06:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-06 11:09:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-06 11:10:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-06 11:20:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-06 11:22:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Lobjet est déjà fermé
The object is already closed [90007-197]
2019-01-06 12:14:06 database: flush
org.h2.message.DbException: Erreur générale: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at fr.mim.gamestoreAPI.database.H2DataBase.connectionToDatabase(H2DataBase.java:53)
	at fr.mim.gamestoreAPI.database.H2DataBase.createDataBase(H2DataBase.java:24)
	at fr.mim.gamestoreAPI.modele.Magasin.<init>(Magasin.java:22)
	at fr.mim.gamestoreAPI.controleurs.JeuControleur.<clinit>(JeuControleur.java:37)
	at fr.mim.gamestoreAPI.TestControleur.testAddJeuExistant(TestControleur.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: Erreur générale: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 52 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 46 more
2019-01-06 12:14:43 database: flush
org.h2.message.DbException: Erreur générale: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at fr.mim.gamestoreAPI.database.H2DataBase.connectionToDatabase(H2DataBase.java:55)
	at fr.mim.gamestoreAPI.database.H2DataBase.createDataBase(H2DataBase.java:24)
	at fr.mim.gamestoreAPI.modele.Magasin.<init>(Magasin.java:22)
	at fr.mim.gamestoreAPI.controleurs.JeuControleur.<clinit>(JeuControleur.java:37)
	at fr.mim.gamestoreAPI.TestControleur.testAddJeuExistant(TestControleur.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: Erreur générale: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 52 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Proprietaire/Desktop/workspace/gamestore-api/database/dataFile.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 46 more
